{"name": "kindly Check login functionality -- @1.2 Credentials", "status": "failed", "statusDetails": {"message": "AssertionError\n", "trace": "  File \"C:\\Users\\Intel\\Documents\\Environments\\behave\\lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"C:\\Users\\Intel\\Documents\\Environments\\behave\\lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\test1.py\", line 35, in step_impl\n    assert False\n"}, "steps": [{"name": "Given the user enters \"user2\" and \"pwd2\"", "status": "failed", "statusDetails": {"message": "AssertionError\n", "trace": "  File \"C:\\Users\\Intel\\Documents\\Environments\\behave\\lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"C:\\Users\\Intel\\Documents\\Environments\\behave\\lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\test1.py\", line 35, in step_impl\n    assert False\n"}, "start": 1670681800451, "stop": 1670681852147}, {"name": "Then the user should be logged in", "status": "skipped", "start": 1670681852150, "stop": 1670681852150}], "parameters": [{"name": "name", "value": "user2"}, {"name": "password", "value": "pwd2"}], "start": 1670681800449, "stop": 1670681852150, "uuid": "3ffba526-e380-42cc-a44f-4fbd9a256aa0", "historyId": "3708ab36af5a368ce7159292a2b07043", "fullName": "User information: kindly Check login functionality", "labels": [{"name": "severity", "value": "normal"}, {"name": "feature", "value": "User information"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}